scripts/translation/lib/RevcheckIgnore.php

```diff
@@ -57,13 +57,6 @@ public static function ignore( $filename ) : bool
         if ( $filename == "reference/datetime/timezones.xml" )
             return true;
 
-        if ( IGNORE_EXTENSIONS_XML )
-            if ( str_ends_with( $filename , '/extensions.xml') ) // track/count backport
-                return true;
-        if ( IGNORE_CHMONLY_DIR )
-            if ( str_starts_with( $filename , 'chmonly/') )      // track/count backport
-                return true;
```

Comment:
Similarly can we hardcode the 4 files?

Reply:
In the same theme of having as few exceptions as possible, this change makes `chmonly/` a normal translatable directory, as it seems to me (albeit the name). This directory only contains the four .xml files, all translatable.

It is no different from `chapters/` or ` faq/` in that matter. And with CHM manuais building again, the files in this directory are being included, untranslated, for all translations.

Reply:
I don't think CHM builds other than English are provided, thus I don't see the point in making those files available for translation. @jimwins do you know if CHM builds can work with translations.

Reply:
We do build CHM files for other languages (not Italian and Ukrainian for some reason). @mumumu does the builds, maybe they'd know if translated files in `chmonly` would be picked up. It seems to me that they should be, but I can't build the CHM files so I can't confirm.

Reply:
I downloaded the chm file for Brazilian Portuguese from https://www.php.net/download-docs.php , extracted it and grep'ed it for `The Windows HTML Help Edition of the PHP Manual`, with a match. So this files are being included on translated chm manual builds. The page indicates builds as recent as 8 Nov 2024, if I ready ir correctly.

Reply:
Update. I now confirmed all 4 files of `chmonly` are included and linked on internal CHM spine and HTML rendered ToC.

Reply:
ACK, then this is fine.